components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/par/OAuth2ParEndpoint.java

```diff
@@ -88,9 +90,11 @@ public Response par(@Context HttpServletRequest request, @Context HttpServletRes
                         MultivaluedMap<String, String> params) {
 
         try {
-            checkClientAuthentication(request);
-            handleValidation(request, params);
             Map<String, String> parameters = transformParams(params);
+            // Wrap the request with the parameters obtained from the PAR endpoint.
```

Comment:
Shall we explain in the comment why this wrapping was needed.

Reply:
Updated the comment in [d17f2ca](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2277/commits/d17f2caa765dfe88db7667e1e853d450a5e8132a)